### PR TITLE
Remove non-OpenEthereum content, fix links and mentions

### LINKED ISSUE #336: Clean up references to whisper
We're planning on moving whisper logic out of the node itself, potentially into its own repo.

Docs should reflect that when it happens.

### PR DESCRIPTION
**Pull request by vorot93**
- Replace Parity/Parity Ethereum with OpenEthereum everywhere.
- Remove non-OpenEthereum (Substrate, Fether), obsoleted (dapps, UI) and deprecated (light client, private transactions, auto updater) content.
- General proofreading

Fixes #336

### CODE REVIEW

File: JSONRPC-parity-module.md [hunk 1]
```diff
@@ -1882,11 +1882,11 @@ Response
   "id": 1,
   "jsonrpc": "2.0",
   "result": [
-    "2017-01-20 18:14:19  Updated conversion rate to Ξ1 = US$10.63 (11199212000 wei/gas)",
-    "2017-01-20 18:14:19  Configured for DevelopmentChain using InstantSeal engine",
-    "2017-01-20 18:14:19  Operating mode: active",
-    "2017-01-20 18:14:19  State DB configuration: fast",
-    "2017-01-20 18:14:19  Starting Parity/v1.7.0-unstable-2ae8b4c-20170120/x86_64-linux-gnu/rustc1.14.0"
+    "2020-05-11 18:14:19  Updated conversion rate to Ξ1 = US$10.63 (11199212000 wei/gas)",
```
Comment (dvdplm) [inline JSONRPC-parity-module.md:1885]: Price was really low on Monday! ;)

File: JSONRPC-parity-module.md [hunk 2]
```diff
@@ -1048,7 +1048,7 @@ None
 
 - `Object` - Information on current version.
     - `hash`:   `Hash` - 20 Byte hash of the current build.
-    - `track`:   `String` - Track on which it was released, one of: `"stable"`, `"beta"`, `"nightly"`, `"testing"`, `"null"` (unknown or self-built).
+    - `track`:   `String` - Track on which it was released, one of: `"stable"`, `"nightly"`, `"testing"`, `"null"` (unknown or self-built).
```
Comment (dvdplm) [inline JSONRPC-parity-module.md:1051]: Not relevant anymore I believe.

File: Getting-Synced.md [hunk 3]
```diff
@@ -2,49 +2,40 @@
...
-State snapshotting, or warp-sync, allows for an extremely fast "synchronization" that skips almost all of the block processing, simply injecting the appropriate data directly into the database. To use a snapshot sync, you first need to download a snapshot.
-
-When using Parity **1.4** or **1.5**, you can just run with:
-
-```bash
-$ parity --warp
-```
-
-... to automatically fetch a recent snapshot from the network, restore from it, and continue syncing. When using Parity **1.6** or **1.7**, `--warp` does nothing as it is enabled by default.
+State snapshotting, or Warp Sync, allows for an extremely fast "synchronization" that skips almost all of the block processing, simply injecting the appropriate data directly into the database. To use a snapshot sync, you first need to download a snapshot.
```
Comment (dvdplm) [inline Getting-Synced.md:13]:
```suggestion
State snapshotting, or Warp Sync, allows for an fast "synchronization" that skips almost all of the block processing, simply injecting the appropriate data directly into the database. To use a snapshot sync, you first need to download a snapshot.
```

File: Fuzz-Testing.md [hunk 4]
```diff
@@ -47,12 +47,12 @@ For details on cargo-fuzz output, check the great write-up [here](https://llvm.o
...
 So, the tools are installed, the project directory is instrumented, and there is an empty fuzzing harness. Now what?
 
-Let's fill in the blanks using a simple test case, fuzzing the keccak hash library used by Parity.
+Let's fill in the blanks using a simple test case, fuzzing the keccak hash library used by OpenEthereum.
 
 First, repeat the steps to initialize a new fuzzing project:
 
 ```
-cd /path/to/parity/util/hash
+cd /path/to/openethereum/util/hash
```
Comment (dvdplm) [inline Fuzz-Testing.md:55]: This doesn't exist in that path. I think this refers to [`keccak-hash`](https://github.com/paritytech/parity-common/tree/master/keccak-hash) but yeah, not sure this fuzzing stuff is still relevant (@ordian do you know?)

File: FAQ.md [hunk 5]
```diff
@@ -13,69 +13,30 @@ libs:
...
 
-The quickest way of syncing Parity Ethereum client with the top of the chain is achieved using Warp sync in combination with `warp-barrier`.
+Read more about [Warp Sync](Warp-Sync-Snapshot-Format).
 
-To make sure you get the most recent snapshot available and warp sync very close to the top of the chain, Parity Ethereum >=v1.10.1 supports the flag `--warp-barrier [NUM]`. Replace [NUM] with [the chain's latest block number](https://stats.parity.io/) **minus at least 10,000 blocks**. Example: if latest block number is 5,633,123 you should launch `parity --warp-barrier 5620000`.
+### What are the OpenEthereum disk space needs and overall hardware requirements?
+Running a node with the standard configuration for the Ethereum Mainnet requires a lot of computer resources. The blockchain download and validation process are particularly heavy on CPU and disk IO. It is therefore recommended to run a full node on a computer with multi-core CPU, 4GB RAM and an SSD drive and at least 200GB free space. Internet connection can also be a limiting factor. A decent DSL connection is required.
```
Comment (dvdplm) [inline FAQ.md:24]:
```suggestion
Running a node with the standard configuration for the Ethereum Mainnet requires a lot of computer resources. The blockchain download and validation process are particularly heavy on CPU and disk IO. It is therefore recommended to run a full node on a computer with multi-core CPU, 8GB RAM and an SSD drive and at least 400GB free space. Internet connection can also be a limiting factor. A decent DSL connection is required.
```

File: Docker.md [hunk 6]
```diff
@@ -163,29 +149,29 @@ $ docker attach --sig-proxy=false 245f312f3f39
 
 Disabling the signal proxy [allows to detach again](http://stackoverflow.com/a/22894096) with `CTRL`+`C`.
 
-### Parity Ethereum Deploy Scripts
+### OpenEthereum Deploy Scripts
 
-A Parity Ethereum deployment script generator is available at [paritytech/parity-deploy](https://github.com/paritytech/parity-deploy). It uses `docker` and `docker-compose`. On Ubuntu systems these will automatically be installed if not already present on the system.
+A OpenEthereum deployment script generator is available at [paritytech/parity-deploy](https://github.com/paritytech/parity-deploy). It uses `docker` and `docker-compose`. On Ubuntu systems these will automatically be installed if not already present on the system.
```
Comment (dvdplm) [inline Docker.md:154]:
```suggestion
A OpenEthereum deployment script generator is available at [paritytech/parity-deploy](https://github.com/openethereum/parity-deploy). It uses `docker` and `docker-compose`. On Ubuntu systems these will automatically be installed if not already present on the system.
```

…but not sure how that has stood up to the test of time.
/cc @denisgranha

File: index.md [hunk 7]
```diff
@@ -1,15 +1,29 @@
...
+- Minimal memory and storage footprint
+- Synchronise in hours, not days with Warp Sync
+- Modular for light integration into your service or product
+
+## Technical Overview
+OpenEthereum's goal is to be the fastest, lightest, and most secure Ethereum client. We are developing OpenEthereum using the sophisticated and cutting-edge **Rust programming language**. OpenEthereum is licensed under the GPLv3 and can be used for all your Ethereum needs.
+
+By default, OpenEthereum runs a JSON-RPC HTTP server on port `:8545` and a Web-Sockets server on port `:8546`. This is fully configurable and supports a number of APIs.
+
+Some help:
```
[the remark this answers is not in the document]
Comment (vorot93) [reply index.md:24]: added

File: _includes/_Sidebar.md [hunk 8]
```diff
@@ -75,80 +62,12 @@
...
-- Whisper
-  - [Whisper](Whisper)
-  - [Overview](Whisper-Overview)
-  - [Whisper PoC #2](Whisper-PoC-2)
-  - [Wire Protocol](Whisper-PoC-2-Wire-Protocol)
 
 ### Hacking
-- [Parity Ethereum Style Guide](Parity-Ethereum-Style-Guide)
-- [Known Issues](https://github.com/paritytech/parity-ethereum/issues) <i class="fa fa-external-link"></i>
+- [Known Issues](https://github.com/openethereum/openethereum/issues) <i class="fa fa-external-link"></i>
```
[the remark this answers is not in the document]
Comment (vorot93) [reply _includes/_Sidebar.md:67]: Yeah, it's the default. Also my rule of thumb is that the shorter the link the better.

File: _includes/_Sidebar.md [hunk 9]
```diff
@@ -1,18 +1,17 @@
...
-### Using Parity Ethereum
-- [Configuring Parity Ethereum](Configuring-Parity-Ethereum)
+### Using OpenEthereum
+- [Configuring OpenEthereum](Configuring-OpenEthereum)
 - [For Mining](Mining)
 - [Wallet Remote Access](Wallet-Remote-Access)
 - [Network Config](Network-Configuration)
 - [Docker](Docker)
 - [CLI Sub-commands](CLI-Sub-commands)
-- [Community Guides](Community-Guides)
```
[the remark this answers is not in the document]
Comment (vorot93) [reply _includes/_Sidebar.md:15]: Exactly the reason why I removed it. IMO outdated content is worse than no docs, and community guides are bound to become stale very quickly.

File: WebAssembly-Home.md [hunk 10]
```diff
@@ -49,14 +49,14 @@ pub fn call() {
...
 
 ## Other languages for contracts
 
-See our [Runtime specification](WebAssembly-Runtime.md). If your languages of choice can be compiled in WebAssembly and can generate bindings for those syscalls, you are good to go.
+If your languages of choice can be compiled in WebAssembly and can generate bindings for those syscalls, you are good to go.
 
 ## Integrating in the existing toolchain
 
-We believe WebAssembly is here to stay, therefore we contribute not only to the Parity to support Wasm contract execution, but to WebAssembly ecosystem as a whole.
+We believe WebAssembly is here to stay, therefore we contribute not only to the OpenEthereum to support Wasm contract execution, but to WebAssembly ecosystem as a whole.
```
[the remark this answers is not in the document]
Comment (vorot93) [reply WebAssembly-Home.md:60]: removed paragraph

File: Basic-Usage.md [hunk 11]
```diff
@@ -2,39 +2,38 @@
...
 
 ### Configuration
 
-Use `parity --help` to find out about available options. To avoid adding options on every run of Parity a [config file](Configuring-Parity.md#config-file). The file can be easily generated using [this tool](https://paritytech.github.io/parity-config-generator/).
+Use `openethereum --help` to find out about available options. To avoid adding options on every run of OpenEthereum a [config file](Configuring-OpenEthereum#config-file).
 
 ### Networking
 
-You can connect to the Ropsten testnet with `parity --chain ropsten` or to the Kovan testnet with `parity --chain kovan`.
+You can connect to the Ropsten testnet with `openethereum --chain ropsten` or to the Kovan testnet with `openethereum --chain kovan`.
```
Comment (denisgranha) [inline Basic-Usage.md:20]: I would rather reference rinkeby and goerli as "modern" testnets

File: CLI-Sub-commands.md [hunk 12]
```diff
@@ -57,15 +56,15 @@ Import blockchain from a file to the given --chain database (default: foundation
...
+### Example: `openethereum import ./export-blocks-file.bin`
 
 ## Account new
 
 Create a new account (and its associated key) for the given --chain (default: foundation).
 
 ### Example:
 ```bash
-parity --chain kovan account new
+openethereum --chain kovan account new
```
Comment (denisgranha) [inline CLI-Sub-commands.md:67]: same, reference a testnet we actually run and know of it's stability

File: Configuring-OpenEthereum.md [hunk 13]
```diff
@@ -212,16 +212,16 @@ Operating Options:
...
         Set the operating mode. MODE can be one of: last - Uses
-        the last-used mode, active if none; active - Parity
-        continuously syncs the chain; passive - Parity syncs
+        the last-used mode, active if none; active - client
+        continuously syncs the chain; passive - client syncs
         initially, then sleeps and wakes regularly to resync;
-        dark - Parity syncs only when the JSON-RPC is active;
-        offline - Parity doesn't sync. (default: last)
+        dark - client syncs only when the JSON-RPC is active;
+        offline - client doesn't sync. (default: last)
```
Comment (denisgranha) [inline Configuring-OpenEthereum.md:224]: I think @dvdplm mentioned this mode persists in db and it makes it run again on that mode? if that's the case, we shouls warn it here IMO

File: JSONRPC-parity-module.md (hunk 1 above)
Comment (denisgranha) [reply JSONRPC-parity-module.md:1885]: hahah, good old times

File: Private-development-chain.md [hunk 14]
```diff
@@ -2,21 +2,21 @@
...
+The default configuration should work fine in most cases. However, it can be customised. You can see the [chain specification for the dev network](https://github.com/openethereum/openethereum/blob/master/ethcore/res/instant_seal.json) in the source repository. You can make a copy locally, modify it and pass it to the `--chain` option. The last section of the file under `accounts` can be modified to add more accounts with lots of Ether. See the [Chain specification](Chain-specification) documentation for more details.
 
 To make multiple transactions confirm at the same time use `--reseal-min-period 0` and to make transactions free use `--gasprice 0`.
 
 
 ## Vagrant
-There is a [Vagrant Box](https://github.com/jesuscript/vagrant-eth-env) that creates and starts a preconfigured Parity Ethereum node running this chain, in case you are looking for a minimal configuration setup.
+There is a [Vagrant Box](https://github.com/jesuscript/vagrant-eth-env) that creates and starts a preconfigured OpenEthereum node running this chain, in case you are looking for a minimal configuration setup.
```
[the remark this answers is not in the document]
Comment (denisgranha) [reply Private-development-chain.md:22]: I haven't tried it but we reference already the docker option so I wouldn't use this one

File: Setup.md [hunk 15]
```diff
@@ -165,10 +165,5 @@ $ cargo run --release --help
 In case you need to persist the blockchain files, keys etc., you should run the image with the `--base-path` option and then mount it, e.g.:
 
 ```bash
-$ docker run --name parity -v /srv/parity:/mnt ethcore/parity:beta --base-path /mnt
+$ docker run --name openethereum -v /srv/openethereum:/mnt openethereum/openethereum:beta --base-path /mnt
```
[the remark this answers is not in the document]
Comment (denisgranha) [reply Setup.md:168]: no, there is no beta image, that should be either :latest or the new :nightly we will add

File: Proof-of-Authority-Chains.md [hunk 16]
```diff
@@ -99,4 +97,4 @@ An example template that can be used for the consensus algorithms:
 }
 ```
 
-The only available algorithm for PoA is [Aura](Pluggable-Consensus.md#aura), other consensus might be added in the future. More details on other chain parameters are available on the [Chain specification](Chain-specification.md) page, there are also [important considerations](Pluggable-Consensus.md#operational-tradeoffs) when setting up your Proof of Authority chain in order to ensure it works reliably.
+The only available algorithm for PoA is [Aura](Pluggable-Consensus#aura), other consensus might be added in the future. More details on other chain parameters are available on the [Chain specification](Chain-specification) page, there are also [important considerations](Pluggable-Consensus#operational-tradeoffs) when setting up your Proof of Authority chain in order to ensure it works reliably.
```
[the remark this answers is not in the document]
Comment (vorot93) [reply Proof-of-Authority-Chains.md:100]: added

File: Secret-Store-Tutorial-1.md [hunk 17]
```diff
@@ -2,20 +2,20 @@
...
 
-- Make sure you have all the [build dependencies](https://github.com/paritytech/parity-ethereum/#build-dependencies).
+- Make sure you have all the [build dependencies](https://github.com/openethereum/openethereum/#build-dependencies).
 - Run
 ```
-$ git clone https://github.com/paritytech/parity
-$ cd parity
+$ git clone https://github.com/openethereum/openethereum
+$ cd openethereum
 $ git checkout beta
```
[the remark this answers is not in the document]
Comment (vorot93) [reply Secret-Store-Tutorial-1.md:14]: fixed

File: Setup.md (hunk 15 above)
[the remark this answers is not in the document]
Comment (vorot93) [reply Setup.md:172]: fixed

File: JSONRPC-parity-module.md (hunk 1 above)
Comment (vorot93) [reply JSONRPC-parity-module.md]: fixed

File: Fuzz-Testing.md (hunk 4 above)
Comment (ordian) [reply Fuzz-Testing.md:55]: this guide specifies how to set up fuzzing from scratch, but it's quite a bit outdated (e.g. nightly is no longer needed)
we should either refer to https://rust-fuzz.github.io/book/cargo-fuzz.html or remove it completely

File: Permissioning.md [hunk 18]
```diff
@@ -239,23 +237,23 @@ More are being implemented.
...
+In OpenEthereum a Validator Set can be specified using a contract implementing a special interface. Thanks to the smart contract definition authorities can be managed according to any rules suitable to the particular application.
 
 ### How it works
-Please see the [Validator Set wiki page](Validator-Set.md).
+Please see the [Validator Set wiki page](Validator-Set).
 
 ## Gas price
-Currently, Parity already allows whitelisting of accounts for zero gas price transactions. In addition to that, a way to specify gas prices per account will be possible soon. This is achieved by managing gas prices in a smart contract and enables one to regulate how much each account has to spend on interactions with the blockchain. Of course, gas prices can also be set to zero.
+Currently, OpenEthereum already allows whitelisting of accounts for zero gas price transactions. In addition to that, a way to specify gas prices per account will be possible soon. This is achieved by managing gas prices in a smart contract and enables one to regulate how much each account has to spend on interactions with the blockchain. Of course, gas prices can also be set to zero.
```
[the remark this answers is not in the document]
Comment (vorot93) [reply Permissioning.md:249]: The whole **Gas price** section?

File: JSONRPC-parity-module.md (hunk 2 above)
Comment (vorot93) [reply JSONRPC-parity-module.md:1051]: RPC call is still there.

File: Fuzz-Testing.md (hunk 4 above)
Comment (vorot93) [reply Fuzz-Testing.md]: Removed

File: Configuring-OpenEthereum.md (hunk 13 above)
Comment (vorot93) [reply Configuring-OpenEthereum.md:224]: I'd rather have this fixed later since ideally this section should be a copy of OpenEthereum's CLI.

**Comment by vorot93** [comment]
@ordian we could always resurrect the deleted content from the git history. But one of the purposes of deprecation is to gracefully steer users away from these features, and working on docs kind of defeats the point IMO.

File: Docker.md [hunk 19]
```diff
@@ -2,86 +2,72 @@
...
 "nightly"
 "stable"
-"v2.0.6"
-"v2.0.7"
-"v2.0.8"
-"v2.1.2"
-"v2.1.3"
-"v2.1.4"
-"v2.2.0-rc6"
+"v3.0.0"
```
Comment (adria0) [inline Docker.md:38]: 3.0.1

File: Docker.md [hunk 20]
```diff
@@ -2,86 +2,72 @@
...
 ```
 
 To get the latest beta release, run:
```
Comment (adria0) [inline Docker.md:41]: The get the lastest release, run:

`docker pull openethereum/openethereum:latest` ?

File: Docker.md [hunk 21]
```diff
@@ -2,86 +2,72 @@
...
 
 ```bash
-$ docker pull parity/parity:v2.1.3
+$ docker pull openethereum/openethereum:v3.0.0
```
Comment (adria0) [inline Docker.md:44]: The get the lastest release, run:

docker pull openethereum/openethereum:latest ?

File: index.md [hunk 22]
```diff
@@ -1,15 +1,30 @@
...
+**Built for mission-critical use**: Miners, service providers, and exchanges need fast synchronisation and maximum uptime. OpenEthereum provides the core infrastructure essential for speedy and reliable services.
+
+- Clean, modular codebase for easy customisation
+- Advanced CLI-based client
+- Minimal memory and storage footprint
+- Synchronise in hours, not days with Warp Sync
+- Modular for light integration into your service or product
+
+## Technical Overview
+OpenEthereum's goal is to be the fastest, lightest, and most secure Ethereum client. We are developing OpenEthereum using the sophisticated and cutting-edge **Rust programming language**. OpenEthereum is licensed under the GPLv3 and can be used for all your Ethereum needs.
```
Comment (adria0) [inline index.md:20]: "sophisticated" word sounds snob to be

**Pull request merged**